src/Pagination/components/Pagination.ts

```diff
@@ -36,9 +36,18 @@ export class Pagination extends Component<PaginationProps, PaginationState> {
 
     render() {
         const { listViewSize, pageSize, hideUnusedPaging } = this.props;
-        const isVisible = !hideUnusedPaging || hideUnusedPaging && listViewSize > pageSize;
-        return createElement("div",
-            { className: classNames("pagination", `${isVisible ? "visible" : "hidden"}`) },
+        const isVisible =
+            !hideUnusedPaging || (hideUnusedPaging && listViewSize > pageSize);
+        return createElement(
+            "div",
+            {
+                className: classNames(
+                    "pagination",
+                    `${isVisible ? "visible" : "hidden"}`
+                ),
+                role: "navigation",
+                "aria-label": "List pagination"
```

Comment:
Translate using `mxui.widget.Grid.a11y.navigation_aria_label`

Reply:
Changed it.